### ISSUE TITLE
[Bug]Run/Build Manifest Merge Errors in a clean install

### ISSUE THREAD

**Issue by Jflima92** [EddyVerbruggen/nativescript-plugin-firebase#450]
I am having the following error when trying to use this plugin in a template, but I guess it will happen in a blank project.

Here's the error I got:
```Execution failed for task ':processF0F1F2DebugManifest'.
> Manifest merger failed : Attribute meta-data#android.support.VERSION@value value=(25.3.1) from [com.android.support:cardview-v7:25.3.1] AndroidManifest.xml:24:9-31
  	is also present at [com.android.support:design:26.0.0-alpha1] AndroidManifest.xml:27:9-38 value=(26.0.0-alpha1).
  	Suggestion: add 'tools:replace="android:value"' to <meta-data> element at AndroidManifest.xml:22:5-24:34 to override.
```

To replicate this issue I've used Sidekick to create a new Drawer template, installed this plugin, copied my configuration files from other project, ran and the error occurs. 
If I remove the plugin the error goes away.

Thanks.

Info:

```
| Component        │ Current version │ Latest version │ Information   │
│ nativescript     │ 3.1.3           │ 3.1.3          │ Up to date    │
│ tns-core-modules │ 3.1.1           │ 3.1.1          │ Up to date    │
│ tns-android      │ 3.1.1           │ 3.1.1          │ Up to date    │
│ tns-ios          │                 │ 3.1.0          │ Not installed │
```

**Comment by Jflima92** [comment]
It was fixed by defining `--compileSdk 25"` argument in the tns run/build.
At least for now it works as workaround.

**Comment by EddyVerbruggen** [comment]
@Jflima92 You may be interested in the fix referenced in #451.

**Comment by Jflima92** [comment]
@EddyVerbruggen Will definitely have a look, thanks!
